docs/content/latest/explore/ysql-language-features/partitions.md

```diff
@@ -54,9 +53,9 @@ CREATE TABLE employees (
 );
 ```
 
-*change_date* represents the date when any type of change occured on the employee record. This date is typically required when generating monthly reports and only the last month's data is queried. To reduce the amount of old data that  needs to be stored, You can reduce the amount of data to store and keep these data for one year, and on the first of every month you remove the data from the previous month. You can partition the `employees` table to meet these requirements. You start by specifying bounds corresponding to the partitioning method and partition key of the `employees` table. This means you create partitions as regular tables and YSQL generates partition constraints automatically based on the partition bound specification every time they have to be referenced.
+*change_date* represents the date when any type of change occurred on the employee record. This date is typically required when generating monthly reports. Assuming that typically only the last month's data is queried often, then the data that is older than one year is removed from the table every month. To simplify this process, you can partition the `employees` table. You start by specifying bounds corresponding to the partitioning method and partition key of the `employees` table. This means you create partitions as regular tables and YSQL generates partition constraints automatically based on the partition bound specification every time they have to be referenced.
```

Comment:
My bad again. 
"Assume that typically only the last month's data is queried often and any data older than a year is removed from the table periodically".

Reply:
Thanks for your suggestion.